examples/resources/grafana_data_source/import.sh

```diff
@@ -1,2 +1,2 @@
-terraform import grafana_data_source.by_integer_id {{datasource id}}
-terraform import grafana_data_source.by_uid {{datasource uid}}
+terraform import grafana_data_source.by_integer_id {{datasource org_id:id}}
+terraform import grafana_data_source.by_uid {{datasource org_id:uid}}
```

Comment:
```suggestion
terraform import grafana_data_source.by_integer_id "{{org_id}}:{{datasource_id}}"
terraform import grafana_data_source.by_uid "{{org_id}}:{{datasource_uid}}"
```

It should look somewhat like templating IMO, otherwise the brackets are confusing

Reply:
Done